apps/callscreen/style/oncall.css

```diff
@@ -284,24 +284,27 @@
     background-image: url('images/call_screen/Ico_call_merge_40x40.png');
   }
 
+  /* When active show the light blue icon */
   .co-advanced-option.active-state .icon.icon-mute,
   .co-advanced-option.active-state .icon.icon-speaker,
-  .co-advanced-option.active-state .icon.icon-bt,
-  .co-advanced-option.active-state:active .icon.icon-on-hold {
+  .co-advanced-option.active-state .icon.icon-bt {
     background-position: 0 4rem;
   }
 
-  .co-advanced-option.active-state .icon.icon-on-hold {
-    background-position: 0 8rem;
+  /* When pressed show the grey icon if the control is enabled */
+  .co-advanced-option:not([disabled]):active .icon.icon-mute,
+  .co-advanced-option:not([disabled]):active .icon.icon-speaker,
+  .co-advanced-option:not([disabled]):active .icon.icon-bt {
+    background-position: 0 0;
   }
 
-  .co-advanced-option:active .icon.icon-mute,
-  .co-advanced-option:active .icon.icon-speaker,
-  .co-advanced-option:active .icon.icon-bt {
-    background-position: 0 0;
+  /* When active show the light blue icon */
+  .co-advanced-option.active-state .icon.icon-on-hold {
+    background-position: 0 8rem;
   }
 
-  .co-advanced-option.active-state:active .icon.icon-on-hold {
+  /* When pressed show the grey icon if the control is enabled */
```

Comment:
Yeah, actually this is quite strange. The light blue active icon is in the middle of the image (4rem) while the grey active icon is at the bottom (8rem) yet these selectors give the right result. I've even checked with the inspector and it works correctly: 8rem gives me the light blue icon and 4rem gives me the grey one. I have no idea what's going on but in principle it shouldn't work yet it does. I'll have a look next Monday before landing.